tests/Utilities/testframework.f90

```diff
@@ -66,53 +66,53 @@ MODULE IRON_TEST_FRAMEWORK
 
   !> Initialise contents of test framework. Must be done before first use.
   SUBROUTINE INITIALISE_TESTS()
-    current_test_name = ""
-    test_err = 0
+    currentTestName = ""
+    testErr = 0
   END SUBROUTINE INITIALISE_TESTS
 
   !> Clean up anything used in test framework and return result. Call after last test.
   SUBROUTINE FINALISE_TESTS(err)
     INTEGER(CMISSIntg), INTENT(OUT) :: err !< The test result, non-zero if failed.
 
-    err = test_err
+    err = testErr
   END SUBROUTINE FINALISE_TESTS
 
   !> Begin test of the given name. Currently doesn't handle nested tests. Must pair with END_TEST
-  SUBROUTINE BEGIN_TEST(test_name)
-    CHARACTER(LEN=*), INTENT(IN) :: test_name !< The name of the test.
+  SUBROUTINE BEGIN_TEST(testName)
+    CHARACTER(LEN=*), INTENT(IN) :: testName !< The name of the test.
 
-    IF (LEN(TRIM(current_test_name)) > 0) THEN
+    IF (LEN(TRIM(currentTestName)) > 0) THEN
       WRITE(unit = error_unit, fmt = '("Called BEGIN_TEST when test in progress. Cannot nest tests. Aborting.")') 
       STOP 1
     ENDIF
     WRITE(unit = output_unit, fmt = '("----------------------------------------")')
-    WRITE(unit = output_unit, fmt = '("Begin test: ",a)') test_name
-    current_test_name = test_name
+    WRITE(unit = output_unit, fmt = '("Begin test: ",a)') testName
+    currentTestName = testName
   END SUBROUTINE BEGIN_TEST
 
   !> End the current test. Test name is retained from prior call to BEGIN_TEST.
   SUBROUTINE END_TEST()
-    IF (LEN(TRIM(current_test_name)) == 0) THEN
+    IF (LEN(TRIM(currentTestName)) == 0) THEN
       WRITE(unit = error_unit, fmt = '("Called END_TEST without call to BEGIN_TEST. Aborting.")')
       STOP 1
     ENDIF
-    WRITE(unit = output_unit, fmt = '("End test: ",a)') TRIM(current_test_name)
-    current_test_name = ""
+    WRITE(unit = output_unit, fmt = '("End test: ",a)') TRIM(currentTestName)
+    currentTestName = ""
   END SUBROUTINE END_TEST
 
   !> Record that an error has occurred.
-  SUBROUTINE set_err()
-    test_err = 1
+  SUBROUTINE setError()
+    testErr = 1
```

Comment:
Subroutine names and types are upper CaMeL case i.e., SetError. Only variable names are lower CaMeL case.